src/Executable.js

```diff
@@ -440,16 +441,22 @@ export default class Executable {
      * Unlike `shouldRetry` this method does in fact still return a boolean
      *
      * @protected
-     * @param {GrpcServiceError} error
+     * @param {Error} error
```

Comment:
Why we need to change here the type of the error? Could we specify it with GrpcServiceError | Error?

Reply:
I get it. Now we are moving one level up, the GrpcServiceError is an instance of Error and now we've got also HttpError. We are passing the parent class.